apps/cookbook/src/app/showcase/range-showcase/range-showcase.component.html

```diff
@@ -0,0 +1,35 @@
+<p>
+  Range is a control that allows the user to select between a range of values
+</p>
+
+<h2>Simple (Default) range with Reactive Form support</h2>
+<p>
+  <cookbook-example-viewer [html]="defaultExample.template">
+    <cookbook-range-default-example #defaultExample></cookbook-range-default-example>
+  </cookbook-example-viewer>
+</p>
+
+<h2>Range with pin and snap step</h2>
+<p>
+  <cookbook-example-viewer [html]="stepExample.template">
+    <cookbook-range-step-example #stepExample></cookbook-range-step-example>
+  </cookbook-example-viewer>
+</p>
+
+<h2>Range with value shown in external component</h2>
```

Comment:
Not sure why this has it's own example, consider removing this and add the "show value" to the other examples instead